WalletWasabi.Tests/UnitTests/PeriodicRunnerTests.cs

```diff
@@ -0,0 +1,82 @@
+using Nito.AsyncEx;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	public class PeriodicRunnerTests
+	{
+		internal class TestRunner : PeriodicRunner
+		{
+			public AutoResetEvent NextRoundAutoResetEvent { get; private set; } = new AutoResetEvent(false);
+
+			public int RoundCounter { get; private set; }
```

Comment:
Should be placed under the constructor.
I don't know why CodeFactor didn't say detect this.